nexus/client.go

```diff
@@ -121,6 +121,19 @@ func NewDefaultClient(baseURL string) *Client {
 	return NewClient(baseURL).Build()
 }
 
+// IsNonScriptOperationsEnabled checks if the target Nexus server has APIs capabilities without relying on Groovy scripts
+// this should be used if the caller is not sure if the server accepts the capabilities offered by this library, since for
+// security reasons and to comply with Nexus Server default policy, it does not support Groovy scripts execution.
+func (c *Client) IsNonScriptOperationsEnabled() (bool, error) {
```

Comment:
I had to think for a while to understand what this checks just by reading its name. If I understood this correctly, we're checking if the beta API is available and we do this by testing if the maven repository API is available (which is part of the beta API). We do this because functionality which would be implemented by Groovy scripts will be performed by the (now) beta APIs.

Can't we just change this to `BetaAPIAvailable` or similar? Or maybe even `NeedGroovy` (though I think this is less clear). Wdyt?

Reply:
Me too, but I don't want to have `beta` nor `groovy` wording attached to the operation name. Maybe `IsScriptsRequired` might be better. wdyt?

Reply:
I'm ok with `IsScriptsRequired`, but something about it not being grammatically correct really bugs me. 

I know that prefixing boolean methods with `is` is a widespread convention which makes it clear to reader, but still... Ugh. I always personally thought omitting the `is` part is usually fine and as readable (e.g., `if ScriptsRequired {`), but I undestand if you want to follow this convention.

But one last thing: why not mention `beta` directly? It's seems like such a precise fit for what this method does. What are you concerned with?

Reply:
`beta` is a keyword that will end soon (when they move everything to the new API), I don't want that in our public interface.

Yeah, I understand your ODC, and I'll respect that: `ScriptsRequired` will be.

Reply:
Oh, I see your point. I was about to say that when that happens, we won't need this function because it will all be on the GA API, but if we ever need a functionality that is not exposed by any Nexus REST API, having function as a long-standing part of our interface will come in handy, even if the implementation differs heavily.

Reply:
And thank you for for respecting my OCD lol :laughing: